proj/hps_accel/gateware/hps_cfu.py

```diff
@@ -88,6 +89,18 @@ def connect_macc(self, m, set, macc, get):
             result_sink.payload.eq(macc.result),
         ]
 
+    def connect_input_store(self, m, set, get, input_store):
+        m.d.comb += [
+            set.sources[Constants.REG_INPUT_NUM_WORDS].connect(
+                input_store.num_words),
+            set.sources[Constants.REG_SET_INPUT].connect(input_store.input),
+            input_store.output[0].connect(get.sinks[Constants.REG_INPUT_0]),
+            input_store.output[1].connect(get.sinks[Constants.REG_INPUT_1]),
+            input_store.output[2].connect(get.sinks[Constants.REG_INPUT_2]),
+            input_store.output[3].connect(get.sinks[Constants.REG_INPUT_3]),
+            input_store.next.eq(get.read_strobes[Constants.REG_INPUT_3]),
```

Comment:
Oooh this was the missing piece I needed to make the flow control from store -> registers work in my design! I forgot about the read_strobes signals.